### ISSUE TITLE
Dashboard view is broken

### ISSUE THREAD

**Issue by softworkz** [refined-github/refined-github#4936]
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Include in this issue:

- [X] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

### Description

![image](https://user-images.githubusercontent.com/4985349/137702330-0b7fdf08-a8d6-4ddf-8308-f88b340937bf.png)

```
Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.
DevTools failed to load source map: Could not load content for chrome-extension://hlepfoohegkhhmjieoechaddaejaokhf/build/refined-github.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://jbmgjfpepenggaeblnmpdbjapdhkjeep/common/browser-polyfill.min.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://jbmgjfpepenggaeblnmpdbjapdhkjeep/common/purify.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

```

Clicking on "More" yields no results

I also have an organization. When I switch the dashboard context to it, I'm seeing the activity for it.
When switching back, it's the same problem again (no results).

Disabling refined-gh + F5 and the results are visible. Re-Enabling: error like in the screenshot.

### Example URL

https://github.com/

### Browser(s) used

Chrome  Version 93.0.4577.82 (Official Build) (64-bit)

**Comment by fregante** [comment]
Is it broken or is it empty? It might be empty because every event is noise: `hide-noisy-newsfeed-events`

**Comment by softworkz** [comment]
Yup, that was it. It's not "noise" though - at least not from my perception.
The entries that are hidden by this feature are messages about commits and releases from others on repos I'm working on (not my own).

I had been using the refined extension for a long time and re-installed these days after short a break (no more than a few months). The problem hadn't existed earlier (exact same setup).

It's ok for me, know that I know how to switch it off, just wondering whether the noise-conditions make sense as a default behavior.

Thanks a lot for your assistance,
sw

**Comment by softworkz** [comment]
Hm, then the only possible explanation would be that I might have encountered this years ago already (my environment and work habits haven't changed much and it was always the same machine which I have replaced recently), changed that option and forgot about it.. 😉 

Thanks again, I could have gone through the options a hundred times without finding the culprit!

Also thanks for the extension, I think it takes an extra amount of passion to develop and maintain this, as you probably often don't get the appreciation that it deserves, because:

- user installs plugin and plays a bit
- user forgets about the installed plugin
- as time goes by, user discovers refined-gh features
  - one after another
  - user thinks: "oh nice, GitHub has introduced a nice new UI feature"
- when accessing GH from another computer or browser
  - user thinks: "something's wrong with GitHub today, they must have a technical problem"
  (happened to me at some point..)

It seems to be a somewhat cursed situation: the only way to get a full understanding of all the little benefits that refined-gh provides, is to uninstall it after using it for weeks or even months..

Only then, you'll be really ready to say: Good Job!
